CNV/gCNV/xhmm/wdl/xhmm.wdl

```diff
@@ -0,0 +1,636 @@
+            centered_filtered_samples = FilterTargetsAndMeanCenter.mean_center_step_excluded_samples,
+            pca_normalized_filtered_samples = FilterAndZScoreCenter.z_score_step_excluded_samples,
+            xhmm_docker = xhmm_docker,
+            xhmm_executable = xhmm_executable,
+            cohort_id = cohort_id
+    }
+
+    call DiscoverCNV {
+        input:
+            normalized_filtered_zcores = FilterAndZScoreCenter.normalized_filtered_zcores,
+            original_depth_filtered = FilterOriginalReadDepth.original_depth_filtered,
+            discover_parameters_file = discover_parameters_file,
+            xhmm_docker = xhmm_docker,
+            xhmm_executable = xhmm_executable,
+            cohort_id = cohort_id
+    }
+
+    call GenotypeCNV {
+        input:
+            normalized_filtered_zcores = FilterAndZScoreCenter.normalized_filtered_zcores,
+            original_depth_filtered = FilterOriginalReadDepth.original_depth_filtered,
+            discover_file = DiscoverCNV.discovered_file,
+            discover_parameters_file = discover_parameters_file,
+            ref_fasta = ref_fasta,
+            ref_fasta_fai = ref_fasta_fai,
+            ref_fasta_dict = ref_fasta_dict,
+            xhmm_docker = xhmm_docker,
+            xhmm_executable = xhmm_executable,
+            cohort_id = cohort_id
+    }
+
+    output {
+        Array[File] coverage_files = CollectCoverage.read_depth_file
+        File merged_depth_file = MergeDepths.merged_depth_file
+        File? extreme_gc_targets = GetExtremeGCContentTargets.extreme_gc_targets
+        File? low_complexity_targets = GetLowComplexityTargets.low_complexity_targets
+        File filtered_centered_counts = FilterTargetsAndMeanCenter.filtered_centered_counts
+        File mean_center_step_excluded_targets = FilterTargetsAndMeanCenter.mean_center_step_excluded_targets
+        File mean_center_step_excluded_samples = FilterTargetsAndMeanCenter.mean_center_step_excluded_samples
+        File panel_of_normals = RunPCA.panel_of_normals
+        File panel_of_normals_sd = RunPCA.panel_of_normals_sd
+        File normalized_counts  = NormalizeCounts.normalized_counts
+        File normalized_filtered_zcores = FilterAndZScoreCenter.normalized_filtered_zcores
+        File z_score_step_excluded_targets = FilterAndZScoreCenter.z_score_step_excluded_targets
+        File z_score_step_excluded_samples = FilterAndZScoreCenter.z_score_step_excluded_samples
+        File original_depth_filtered = FilterOriginalReadDepth.original_depth_filtered
+        File discovered_file = DiscoverCNV.discovered_file
+        File genotyped_vcf_file = GenotypeCNV.genotyped_vcf_file
+    }
+}
+
+task CollectCoverage {
+    File bam
+    File bam_idx
+    File ref_fasta
+    File ref_fasta_fai
+    File ref_fasta_dict
+    File interval_list
+
+    #Runtime parameters
+    String xhmm_docker
+    Int? mem_gb
```

Comment:
Another note: currently, unless you expose these task-level parameters at the workflow level (e.g., expose `mem_gb_for_collect_coverage`), you will be unable to access them if you wrap this WDL as a subworkflow (e.g, in a larger validation WDL that runs this XHMM WDL).  Hopefully this will change soon, but just something to be aware of.